atomisticparsers/lammps/parser.py

```diff
@@ -780,22 +781,29 @@ def get_traj_files(self):
         return [os.path.join(self.maindir, f) for f in traj_files]
 
     def get_data_files(self):
+        def check_file_header(file_path, regex_pattern):
+            header_size = 1024
+            file_path = f"{self.maindir}/{file_path}"
+            print(file_path)
+            with open(file_path, "rb") as file:
```

Comment:
maybe good to put a try here to open file, and put re.search out of the open context by initializing file_header_str = ''

Reply:
please check again.